tests/integration_tests/test_visualization.py

```diff
@@ -326,6 +330,94 @@ def test_visualization_compare_classifiers_from_prob_csv_output_saved(
             print("Error: %s - %s." % (e.filename, e.strerror))
 
 
+def test_visualization_compare_classifiers_from_prob_csv_output_saved_api(
+        csv_filename
+):
+    """Ensure pdf and png figures from the experiments can be saved.
+
+    Probabilities are loaded from csv file.
+    :param csv_filename: csv fixture from tests.fixtures.filenames.csv_filename
+    :return: None
+    """
+    input_features = [
+        text_feature(vocab_size=10, min_len=1, representation='sparse'),
```

Comment:
remove the text feature, kep only the category one as it is much faster for test purposes, and reduce it's vocab to 2.